specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
```

Comment:
Because you would do `node app.js --files specs/data/transactions/may/transactions.csv`